Database.txt

```diff
@@ -3,3 +3,4 @@ null
 null    
 null    
 null    
+null    
```

Comment:
Do we need this file? If not, can we just delete it?

Reply:
yea sure thing